1_fetch/src/get_nwis_data.R

```diff
@@ -34,6 +34,26 @@ get_nwis_daily_data <- function(site_num, outdir, parameterCd, startDate, endDat
   return(filepath)
 }
 
+get_daily_flow_log <- function(files_in, file_out) {
+  message(paste('generating log for dataRetrieval daily flow request'))
+  daily_flow_list <- map(files_in, read_csv, 
+              col_types = cols(agency_cd = col_skip(), 
+                               site_no = col_character(), 
+                               Date = col_date(format = "%Y-%m-%d"), 
+                               discharge = col_double(), 
+                               discharge_cd = col_character()))
+  daily_flow_df <- bind_rows(daily_flow_list)
+  daily_flow_log <- daily_flow_df %>%
+    group_by(site_no) %>%
+    summarize(num_days = n(), 
```

Comment:
Could you add a column for number of provisional or estimated days? Maybe `num_Pe` for the name